src/autorest/tsconfig.json

```diff
@@ -10,7 +10,6 @@
     "declaration": true,
     "stripInternal": true,
     "lib": [
-      "es5",
```

Comment:
that probably didn't fix anything, but is there any effect of this? shouldn't `es6` fully capture that? (not sure about the `es2015.` stuff)